### DISCUSSION TITLE
Adding `with_attributes` context propagation to pg instrumentation?

### DISCUSSION THREAD

**Discussion by yoheyk** [open-telemetry/opentelemetry-ruby#1341] [General]
I'd like to have the `with_attributes` context propagation in the `pg` instrumentation also.
Currently, we have it in `Redis`, `HTTP::ClientContext` and `mysql2`: https://github.com/open-telemetry/opentelemetry-ruby/pull/1175
It's going to be the forth one so still under the threshold of duplication 😄 : https://github.com/open-telemetry/opentelemetry-ruby/pull/1157#issuecomment-1089070239
How does this sound?

**Comment by ahayworth** [comment]
@yoheyk I think we would welcome such a PR. 😄

**Reply by yoheyk** [reply]
will do!
